packages/common/src/content/_internal.ts

```diff
@@ -89,8 +82,178 @@ const MAX_SIZE = 5000000;
  * @param requestOptions Hub Request Options (including whether we're in portal)
  * @returns
  */
-export const isProxiedCSV = (item: IItem, requestOptions: IHubRequestOptions) =>
-  !requestOptions.isPortal &&
+export const isProxiedCSV = (
+  item: IItem,
+  requestOptions?: IHubRequestOptions
+) =>
+  !isPortal(requestOptions) &&
   item.access === "public" &&
   item.type === "CSV" &&
-  item.size <= MAX_SIZE;
+  item.size <= 5000000;
+
+export const getHubRelativeUrl = (
+  type: string,
+  identifier: string,
+  typeKeywords?: string[]
+): string => {
+  // solution types have their own logic
+  let contentUrl = getSolutionUrl(type, identifier, typeKeywords);
+  if (!contentUrl) {
+    const family = getFamily(type);
+    const pluralizedFamilies = [
```

Comment:
these don't seem pluralized?

Reply:
should be `const familiesWhoseRouteNamesAreThePluralizedFamily`